sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/AllTransactionView.kt

```diff
@@ -57,6 +57,22 @@ internal class AllTransactionView(
                 AllTransactionViewDefinition.COLUMN_INTEGER_MINED_HEIGHT
             )
 
+        /**
+         * Get all sent, unmined transactions that are still within the expiry window
+         *
+         * Requested selection should look like this:
+         * mined_height IS NULL AND expiry_height > ? AND account_balance_delta < 0
+         */
+        private val SELECTION_TRX_RESUBMISSION =
+            String.format(
+                Locale.ROOT,
+                // $NON-NLS
+                "%s IS NULL AND %s > ? AND %s < 0",
```

Comment:
maybe create a constant to keep up with this convention in this file

Reply:
The constant added. Thank you.